File: github/modul 6/Bahasa C/PRAK603-2410817320010-RabiahRiskaAmaliah.c
#include <stdio.h>
int main() {
    int a, b;
    scanf("%d %d", &a, &b);
    if (a != b) {printf("Jumlah tidak sama"); return 1;}

    int barisa[a], barisb[b];
    for (int d = 0; d < a; d++) {scanf("%d", &barisa[d]);}
    for (int d = 0; d < b; d++) {scanf("%d", &barisb[d]);}
    for (int d = 0; d < a; d++) {printf("%d ", barisa[d] * barisb[d]);}
    return 0;
}

File: github/modul 6/Bahasa py/PRAK603-2410817320010-RabiahRiskaAmaliah.py
a, b = map(int, input ().split())

if a!=b: 
    print("Jumlah tidak sama") 
    exit(1)

barisa = list(map(int, input().split()))
barisb = list(map(int, input().split()))

for d in range(a):
    print(barisa[d] * barisb[d], end=" ")